games/urls/achievements.py

```diff
@@ -0,0 +1,12 @@
+# pylint: disable=C0103
+from __future__ import absolute_import
+from django.conf.urls import url
+from games.views import achievements as views
```

Comment:
the `games` app is getting a bit huge (I'm already considering moving the installers in their own app), maybe to would be preferable to do the same for achievements / stats (they both could live in the same app)